src/command.rs

```diff
@@ -39,7 +39,7 @@ const INITIAL_VERSION: Version = Version::new(0, 1, 0);
 const BUFFRS_TESTSUITE_VAR: &str = "BUFFRS_TESTSUITE";
 
 /// Initializes the project
-pub async fn init(kind: PackageType, name: Option<PackageName>) -> miette::Result<()> {
+pub async fn init(kind: Option<PackageType>, name: Option<PackageName>) -> miette::Result<()> {
```

Comment:
Would it make sense to have separate `init_package` and `init_consumer` here, due to the different intent and manifests they write? It can quickly get unwieldy when we have a lot of optional arguments which change the semantics of the function (can close this, just an idea).